### ISSUE TITLE
Do I have your blessing to make an official Zapier for Mastodon bots? 

### ISSUE THREAD

**Issue by switch-to-gitlab** [mastodon/mastodon#18176]
### Pitch

Hello, 

I made a Zapier Zap for making bots, which I'm currently using to make local news bots for my geographically local mastodon instance. 

In order to publish the Zap so that everyone can use it, I need to be able to say that I am an "employee or contractor" for Mastodon. 

So... the long and the short of it is... I recall you hiring me for $0.00 to make a Zapier bot. Is that right? :) 

### Motivation

Having reliable bots for things like crossposting and following news is good for Mastodon, I think, and I'd like it to be easily open and accessible to people who aren't programmers, and even people who are programmers could benefit from the free and reliable hosting options.

**Comment by durablenapkin** [comment]
I agree with your motivation, particularly adding Mastodon to tools like Zapier, Hootsuite and Buffer, to hopefully have individuals/entities who use these tools to have a presence in the fediverse.

**Comment by FarisZR** [comment]
Since the Devs aren't active on this, can you share the code?
so anyone can create a custom module to use until mastodon is official supported?
